app/src/main/java/com/example/alpacamusclemaintenance/ui/HomeFragment.kt

```diff
@@ -58,7 +57,6 @@ class HomeFragment : Fragment(), Injectable {
         .get(HomeViewModel::class.java)
     homeViewModel
       .data
-      .subscribeOn(Schedulers.io())
```

Comment:
TODO: App works without this code but I don't know which thread is used. Look into it. Does it exist any default subscribing thread? 🤔